lib/scan.ts

```diff
@@ -24,12 +24,12 @@ export async function scan(options: Options): Promise<PluginResponse> {
     debug.enabled = !!options?.debug;
     debug('options %o \n', options);
     const extractionDepthLimit =
-      options['--max-depth'] !== undefined
-        ? options['--max-depth']
+      options['max-depth'] !== undefined
+        ? options['max-depth']
```

Comment:
I'm curious, maybe we should make use of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty ? pros and cons?